app/src/main/java/com/mathewsachin/fategrandautomata/util/ScriptMessages.kt

```diff
@@ -142,6 +144,12 @@ class ScriptMessages @Inject constructor(
             )
         }
 
+        if (ceDropCount > 0) {
+            appendLine(
+                context.getString(R.string.ces_dropped, ceDropCount)
```

Comment:
ces_dropped needs to be added to localized.xml

Reply:
Oof ill take a look, thanks!

Reply:
The main one at https://github.com/Fate-Grand-Automata/FGA/blob/master/app/src/main/res/values/localized.xml is the most important one.

Without that, the app won't compile.

Reply:
Small question, I reused this message:

https://github.com/Fate-Grand-Automata/FGA/blob/master/app/src/main/res/values/localized.xml#L292

That should cover the message or should I add another one for this toast specifically? I'm willing to do it either way.

Reply:
Ah sorry, didn't notice that you reused an old one.